cip/CIP2016-06-14-Harmonize-comparability-orderability-equality-and-equivalence.adoc

```diff
@@ -0,0 +1,233 @@
+= CIP2016-06-14 - Harmonize Comparability, Orderability, Equality, and Equivalence
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>, Stefan Plantikow <stefan.plantikow@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP intends to introduce and formalise two new concepts, *comparability* and *orderability*, and defines their relationship to the already existing concepts of *equality* and *equivalence*.
+Comparability revolves around the inequality operators (`>`, `<`, `>=`, `<=`), and defines the semantics they need to properly determine how two values compare to each other.
```

Comment:
This - and operability - are good definitions. I think, though, the abstract may not be best-placed for these definitions. I would just mention here that all 4 concepts are harmonized and defined more precisely (as well as your "in summary" sentence), and then have a `Definitions` section as the first sub-section under `Proposal`.
This is to aid the 'flow' of the document